src/package-vsix.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Package individual built-in VS Code extensions in .vsix packages
+ * The .vsix packages end-up in <repo root>/out
+ */
+// @ts-check
+const fs = require('fs-extra')
+const os = require('os');
+const yargs = require('yargs');
+const { root, out, extensions, run, vscode } = require('./paths.js');
+
+const { tag } = yargs.option('tag', {
+    choices: ['latest', 'next']
+}).demandOption('tag').argv;
+
+const repository = {
+    "type": "git",
+    "url": "https://github.com/theia-ide/vscode-builtin-extensions"
+};
+
+// bump to publish
+let version = '1.39.1';
```

Comment:
Does it mean that we pull a tag commit from VS Code corresponding to this version?

I would prefer to keep independent versions generically and have a mechanism later which can publish for tagger version from VS Code later. Otherwise we will get in version conflicts eventually.

Reply:
> Does it mean that we pull a tag commit from VS Code corresponding to this version?

No, just playing around with versions. I am working off my fork ATM, to minimize risks of accidentally publishing / adding publish commits in this repo, so this is really WIP. The plan is to come-back to the current master versioning scheme before merging here.

Reply:
sounds good